infrastructure/resource_group_level/modules/function-number-generator.bicep

```diff
@@ -0,0 +1,161 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param storageAccountName string
+
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+  APIM_TFS_KEY: 'test-value'
+  APIM_TFS_VALUE: 'test-value'
+  APIM_TFS_URL: 'test-value'
+  APIM_MDM_KEY: 'test-value'
```

Comment:
Note that the `APIM_MDM_*` keys were not in the CLI script, but should be. @abhi-markan will add them in his own PR.

Reply:
this should be added now